Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/ExchangeManagerServiceImpl.java

```diff
@@ -148,18 +190,14 @@ public boolean pingService(ConnectionEndpoint connEndpoint) {
         return false;
     }
 
-    @Override
-    public boolean isRefreshLocked() {
-        return exchangeManager.isRefreshLocked();
+    private static CONNECTClient<EntityExchangeManagementPortType> getClient() throws Exception {
```

Comment:
Do we really want to create a new client each and every time we call getClient? should we cache this into some sort of singleton, or alternative?